service/history/queues/executable.go

```diff
@@ -280,6 +280,11 @@ func (e *executableImpl) HandleErr(err error) (retErr error) {
 		return nil
 	}
 
+	if err == consts.ErrTaskVersionMismatch {
```

Comment:
Plz add a test for this error in executable_test.go

Reply:
Added